migration_average_scores.sql

```diff
@@ -0,0 +1,19 @@
+-- Migration: Add match tracking for average score calculation
```

Comment:
you can delete these. there's nothing to migrate because there is nothing in the db